### PR TITLE
Fixes #13241 (cakehat does not produce light)

### LINKED ISSUE #13241: Cakehat still doesn't produce light
@bgobandit

### PR DESCRIPTION
**Pull request by swankcookie**
fixes #13241 (for mention)
:cl: Swankcookie
bugfix: Cakehat now creates light
tweak: Makes cakehat child of hardhat
/:cl:

### CODE REVIEW

**Comment by swankcookie** [comment]
i fixed the conflict

**Comment by duncathan** [comment]
What a fantastic way to fix it :+1:

**Comment by swankcookie** [comment]
actually? or sarcasm?

**Comment by KorPhaeron** [comment]
@swankcookie 

You forgot to make it do 15 burn damage still please fix this

**Comment by KorPhaeron** [comment]
it also won't start fires anymore, we are missing crucial cakehat functionality in this PR

**Comment by Zilenan91** [comment]
the 15 burn is crucial

**Comment by Jordie0608** [comment]
Merge conflicts, map merger errors and remember to capitalize your changelog entries.

**Comment by MrStonedOne** [comment]
I really don't like how this was done,

A map change and repathing when it's not even what fixed the issue, changing the lum to 2 is what fixed the issue.

**Comment by bgobandit** [comment]
maybe if people pathed shit usefully in the first place :^)

**Comment by optimumtact** [comment]
@MrStonedOne I don't see why you would be opposed to that.

As long as the maps are updated there's zero reason to worry.

I'd rather people worry about making code better than concern themselves with caring about things being repathed.

**Comment by swankcookie** [comment]
closing this as i have no idea how too edit maps and i am very confused.

**Comment by swankcookie** [comment]
Majorly overthinking it.  Thanks @optimumtact

**Comment by newfren** [comment]
Please make sure it does 15 burn when lit for real though, makes it great for fighting carp and shit.

Also the fire starting thing too.

**Comment by swankcookie** [comment]
it does

**Comment by newfren** [comment]
:+1:

**Comment by swankcookie** [comment]
:+1:

File: code/modules/clothing/head/misc_special.dm [hunk 1]
```diff
@@ -67,23 +66,20 @@
...
+    SSobj.processing |= src
 
+/obj/item/clothing/head/hardhat/cakehat/turn_off()
+    ..()
+    force = 0
+    damtype = BRUTE
+    SSobj.processing -= src
 
+/obj/item/clothing/head/hardhat/cakehat/is_hot()
+ 	return on * heat
```
Comment (duncathan) [inline code/modules/clothing/head/misc_special.dm]: You indented with spaces from line 70 onward. Make sure you use tabs. I must've missed this earlier, sorry.

Comment (swankcookie) [inline code/modules/clothing/head/misc_special.dm]: oh shit my bad, I'll fix it when i get back to my PC. Thanks.

**Pull request closed without merging**